components/button/README.md

```diff
@@ -32,34 +54,24 @@ To make your `d2l-button` accessible, use the following properties when applicab
 | `aria-haspopup` | [Indicate clicking the button opens a menu](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-haspopup). Example: [d2l-dropdown](https://github.com/BrightspaceUI/core/blob/master/components/dropdown/dropdown-opener-mixin.js#L46). |
 | `description` | Use when text on button does not provide enough context. |
 
-## d2l-button-subtle
+## Subtle Button [d2l-button-subtle]
```

Comment:
Subtle button and icon button have the `hAlign` property which aligns them with text. Should there be a separate demo that shows that behaviour (non-interactive), since playing around with it in the interactive demo wouldn't really show that (though we could also add some text in there to have that be demonstrated, it just could clutter up the demo)?

Reply:
I'm not too worried about the demo always being able to exercise every single property... a simple demo that makes it easy to understand the basic usage is nice.

Then we can decide if it's worth adding a separate demo just for the horizontal alignment stuff.